src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -67,6 +69,17 @@ public Controls(Subsystems s) {
 		if (INTAKE_ENABLED) {
 			bindIntakeControls();
 		}
+		if (DRIVEBASE_ENABLED && LAUNCHER_ENABLED && INTAKE_ENABLED) {
+			// temporary controls, not sure what drive team wants
+			driveController
+					.leftBumper()
+					.whileTrue(
+							new FullTargetCommand(
+									s.launcherSubsystem,
+									s.intakeSubsystem,
+									s.drivebaseSubsystem,
+									() -> driveController.rightBumper().getAsBoolean()));
```

Comment:
As a side note, `driveController.rightBumper()` is a `Trigger`, which implements `DoubleSupplier`, so you can pass `driveController.rightBumper()` instead of `() -> driveController.rightBumper().getAsBoolean()`.